src/snowflake/snowpark/functions.py

```diff
@@ -988,7 +1044,15 @@ def sha2(e: ColumnOrName, num_bits: int) -> Column:
 
 
 def hash(e: ColumnOrName) -> Column:
-    """Returns a signed 64-bit hash value. Note that HASH never returns NULL, even for NULL inputs."""
+    """
+    Returns a signed 64-bit hash value. Note that HASH never returns NULL, even for NULL inputs.
+
+    Examples::
+
+        >>> df = session.create_dataframe([[10, "10"]], schema=["a", "b"])
```

Comment:
can we also add a hash for null input.

Reply:
Will do. In fact I realized that this also accept multiple arguments so I'll also add support.